bundle/language_pt.properties

```diff
@@ -410,6 +410,8 @@ angal.patient.nextkin                                                       = Pa
 angal.patient.note                                                          = Nota
 angal.patient.parenttogether                                                = Pais juntos
 angal.patient.patientphoto                                                  = Foto de paciente
+angal.patient.patientphoto.error.title                                      = Erro de foto do paciente
+angal.patient.patientphoto.error.filenotfound                               = A foto do paciente n�o foi encontrada.
```

Comment:
There's no need to provide translations as they will come from Transifex once merged on develop, thanks